modules/twinklewarn.js

```diff
@@ -934,6 +934,10 @@ Twinkle.warn.messages = {
 			label: 'Not using edit summary',
```

Comment:
```suggestion
			label: 'New user not using edit summary',
```

Maybe given the divide, wouldn't hurt to be explicit?